spec/dummy/config/application.rb

```diff
@@ -19,9 +19,9 @@ def rails_version_between?(version_1, version_2)
       )
     end
 
-    config.load_defaults 5.1 if rails_version_between?('5.1', '5.2')
-    config.load_defaults 5.2 if rails_version_between?('5.2', '6.0')
-    config.load_defaults 6.0 if rails_version_between?('6.0', '6.1')
-    config.load_defaults 6.1 if rails_version_between?('6.1', '6.2')
+    config.load_defaults 7.0 if rails_version_between?('7.0', '7.1')
+    config.load_defaults 7.1 if rails_version_between?('7.1', '7.2')
+    config.load_defaults 7.2 if rails_version_between?('7.2', '8.0')
+    config.load_defaults 8.0 if rails_version_between?('8.0', '8.1')
```

Comment:
```suggestion
    config.load_defaults Rails::VERSION::STRING.to_f
```

We can also delete `rails_version_between?`